example/lib/registration_view.dart

```diff
@@ -1,13 +1,12 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
 import 'package:flutter_background_geolocation_example/advanced/util/dialog.dart'
     as util;
 import 'package:shared_preferences/shared_preferences.dart';
 
 import 'package:flutter_background_geolocation/flutter_background_geolocation.dart'
     as bg;
 
-import './config/env.dart';
+import './config/ENV.dart';
```

Comment:
There was a mismatch between the filename on disk and other references. I simply renamed the file. This change can be ignored.